site/docs/getting-started/installation.md

```diff
@@ -1,63 +1,43 @@
+## Get started with the Flow web application
 
-If you're using the Flow web application, see the [guide](../guides/create-dataflow.md) to get started.
-:::
-
-Flow includes a [**devcontainer**](https://code.visualstudio.com/docs/remote/containers), which provides a nice development experience using a self-contained Docker-based environment. This is an easy way to get a great development experience using Flow, with auto-completion and an ideal setup for your catalog. There are currently two ways to set this up: virtually, using GitHub Codespaces, and locally, using VS Code and Docker on your machine.
-
-## Using GitHub Codespaces
-
-[GitHub codespaces](https://github.com/features/codespaces) provides VM-backed, portable development environments that are ideal for getting started with Flow in minutes. Currently, Codespaces is available to GitHub Teams and Enterprise customers, as well as individuals enrolled in the beta. If you have access, this is the preferred method — setting up a devcontainer in Codespaces is much quicker than doing so locally.
+You can sign up to get started as a Flow trial user by visiting the web application [here](https://go.estuary.dev/dashboard).
 
-Visit the [Flow Template repository](https://github.com/estuary/flow-template), click **Code**, and choose **New Codespace**.
+Once you've signed up with your personal information, an Estuary team member will be in touch to activate your account and discuss your business use-case, if applicable.
 
-The VM spins up within a minute or two, and you can immediately begin developing and testing. The template includes a PostgreSQL database for this purpose.
+## Get started with the Flow CLI
 
-## Using Visual Studio Code locally
+After your account has been activated through the [web app](#get-started-with-the-flow-web-application), you can begin to work with your data flows from the command line.
+This is not required, but it enables more advanced workflows or might simply be your preference.
 
-If you don't have access to Codespaces, or prefer local development, use this method to create a local environment.
+Flow has a single binary, flowctl. You can download it [here](https://go.estuary.dev/flowctl).
 
-Download and install the following prerequisites:
+[Learn more about using flowctl.](../concepts/flowctl.md)
 
-* [Docker](https://www.docker.com/get-started)
-* [VS Code](https://code.visualstudio.com)
-* VS Code [Remote Containers extension](https://code.visualstudio.com/docs/remote/containers)
+## Self-hosting Flow
 
-#### Create a Git repository from the Flow Template <a href="#create-a-git-repository-from-the-flow-template" id="create-a-git-repository-from-the-flow-template"></a>
+The Flow runtime is available for non-commercial use under the [Business Source License](https://github.com/estuary/flow/blob/master/LICENSE-BSL). It's possible to self-host Flow using a cloud provider of your choice.
```

Comment:
nit: I think we can remove "for non-commercial use". The license does allow commercial use, as long as you don't try to run something like a Flow SAAS.